### PR TITLE
CLI: Added tool to get application by ID

### PR DESCRIPTION
**Pull request by zlamalp**
- Added tool to print result of getApplicationById.
- Added partial object definition for Application.
- Added getApplicationById to CLI API.

### CODE REVIEW

File: perun-cli/Perun/beans/Application.pm [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+sub fromHash
+{
+    my $application = Perun::Common::fromHash(@_);
+    my $vo = $application->{_vo};
+    $vo = Perun::beans::Vo::fromHash("Perun::beans::Vo", $vo);
+    my $group = $application->{_group};
+    $group = Perun::beans::Group::fromHash("Perun::beans::Group", $group);
+    my $user = $application->{_user};
+    $user = Perun::beans::Group::fromHash("Perun::beans::User", $user);
```
Comment (metodej) [inline perun-cli/Perun/beans/Application.pm:19]: I think there should be user
```suggestion
    $user = Perun::beans::User::fromHash("Perun::beans::User", $user);
```

File: perun-cli/Perun/beans/Application.pm [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    my $self = shift;
+
+    return $self->{_extSourceName};
+}
+
+sub getUserDisplayName
+{
+    my $self = shift;
+    my $user = $self->{_user};
+    $user = Perun::beans::Group::fromHash("Perun::beans::User", $user);
```
Comment (metodej) [inline perun-cli/Perun/beans/Application.pm:140]: Same as above
```suggestion
    $user = Perun::beans::User::fromHash("Perun::beans::User", $user);
```

**Review by metodej — CHANGES_REQUESTED** [review]

File: perun-cli/Perun/beans/Application.pm (hunk 1 above)
Comment (zlamalp) [reply perun-cli/Perun/beans/Application.pm]: Fixed.

File: perun-cli/Perun/beans/Application.pm (hunk 2 above)
Comment (zlamalp) [reply perun-cli/Perun/beans/Application.pm]: Fixed

**Review by metodej — APPROVED** [review]

**Pull request merged**